README.md

```diff
@@ -243,7 +243,7 @@ _path/to/myapp/linkerd.yml_, you could start linkerd in docker with
 the following command:
 
 ```
-$ docker run -p 4140:4140 -v path/to/myapp:/myapp -w /myapp buoyantio/linkerd:0.0.10-SNAPSHOT linkerd.yml
+$ docker run -p 4140:4140 -v /absolute/path/to/myapp:/myapp -w /myapp buoyantio/linkerd:0.0.10-SNAPSHOT linkerd.yml
```

Comment:
(apologies in advance for bike shedding on a 10-character change)

I actually think this example would be clearer if you volume mounted the file instead of the whole directory, and got rid of the -w flag.  So...

    $ docker run -p 4140:4140 -v /absolute/path/to/myapp/linkerd.yml:/etc/linkerd.yml buoyantio/linkerd:0.0.10-SNAPSHOT /etc/linkerd.yml